internal/credential/credential.go

```diff
@@ -134,3 +141,9 @@ type SshPrivateKey interface {
 	PrivateKey() PrivateKey
 	PrivateKeyPassphrase() []byte
 }
+
+// Kv is a credential containing a set of key-value pairs.
+type Kv interface {
```

Comment:
Is this interface used anywhere

Reply:
I didn't use the interface anywhere, so I will drop it.